Automattic/wp-calypso
# A4A: Implement the banner section for the new migrations page.

This PR implements the banner section on the new migrations page.

<img width="1425" alt="Screenshot 2024-10-29 at 4 22 47 PM" src="https://github.com/user-attachments/assets/779020fe-1fa7-48e8-98bd-c552e692d163">

Depends on https://github.com/Automattic/wp-calypso/pull/95788
Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1375 

## Proposed Changes

* The new section utilizes the `SimpleList` component, which I have relocated to the global shared folder, as it is now being used in various sections beyond the Marketplace.
* Implement the Banner based on the new design. Add a new background image for the PageSection to match the new design.

## Why are these changes being made?

* This implements the banner for the new migrations page, which will be the first element visible.

## Testing Instructions

* Use the A4A live link and go to the `/migrations` page.
* Confirm you see the banner and it matches the design.
* Test that the banner is responsive for any device screen.

## Pre-merge Checklist
